hapic/infos.py

```diff
@@ -1,2 +1,2 @@
 # -*- coding: utf-8 -*-
-__version__ = '0.43'
+__version__ = '0.46'
```

Comment:
Note: this diff because i (i don't remember when) merge master into my develop branch.